Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -233,54 +220,30 @@ param (
     {
         Create-DestinationDirectory -path $destinationNetworkPath
     }
-    
-    $netExeCommand = Get-NetExeCommand
 
     if($machineShare)
     {
-        $command = "$netExeCommand use `"$machineShare`""
-        if($userName)
-        {
-            $command += " /user:`'$userName`' `'$($password -replace "['`]", '$&$&')`'"
-        }
-        $command += " 2>&1"
-        
-        $dtl_mapOut = iex $command
-        if ($LASTEXITCODE -ne 0) 
-        {
-            $errorMessage = (Get-VstsLocString -Key "WFC_FailedToConnectToPathWithUser" -ArgumentList $machineShare, $($credential.UserName)) + $dtl_mapOut
-            ThrowError -errorMessage $errorMessage -fqdn $fqdn
-        }
+        New-PSDrive -Name "WFCPSDrive" -PSProvider FileSystem -Root $destinationNetworkPath -Credential $psCredentialObject -ErrorAction 'Stop'
     }
-
-    try
+    
```

Comment:
use try catch and get the logs in exception case, will help us to debug if required

Reply:
yes, that sounds right. I'll fix it.